src/networkremote/client.cpp

```diff
@@ -0,0 +1,65 @@
+#include "client.h"
+#include "core/application.h"
+
+NetworkRemoteClient::NetworkRemoteClient(Application *app, QObject *parent)
+  : QObject(parent),
+    app_(app),
+    incomingMsg_(new NetworkRemoteIncomingMsg(this)),
+    outgoingMsg_(new NetworkRemoteOutgoingMsg(app, this))
+{
```

Comment:
{ should be moved to the previous line.
Same for all the added code

Reply:
Done